tests/python/relay/aot/test_crt_aot.py

```diff
@@ -0,0 +1,247 @@
+    x2 = x1 + relay.const(1.0)
+    x3 = x2 + relay.const(1.0)
+    x4 = x3 + relay.const(1.0)
+    out = relay.Tuple([x1, relay.Tuple([relay.Tuple([x2, x3]), x4])])
+    func = relay.Function([x], out)
+
+    x_data = np.random.uniform(size=(10,)).astype(np.float32)
+    inputs = {"x": x_data}
+    output_list = generate_ref_data(func, inputs)
+    input_list = [x_data]
+    compile_and_run(func, input_list, output_list)
+
+
+def test_tuple_getitem():
+    func = relay.Function([], relay.TupleGetItem(relay.Tuple([relay.const(1), relay.const(2)]), 0))
+    output_list = generate_ref_data(func, {})
+    input_list = []
+    compile_and_run(func, input_list, output_list)
+
+
+def test_id():
+    x = relay.var("x", "float32")
+    ident = relay.Function([x], x)
+    one = np.array(1.0, "float32")
+    inputs = {"x": one}
+    output_list = generate_ref_data(ident, inputs)
+    input_list = [one]
+    compile_and_run(ident, input_list, output_list)
+
+
+def test_add_const():
+    two = relay.add(relay.const(1), relay.const(1))
+    func = relay.Function([], two)
+    output_list = generate_ref_data(func, {})
+    input_list = []
+    compile_and_run(func, input_list, output_list)
+
+
+def test_mul_param():
+    x = relay.var("x", shape=(10, 10))
+    y = relay.var("y", shape=(1, 10))
+    func = relay.Function([x, y], relay.multiply(x, y))
+    x_data = np.random.rand(10, 10).astype("float32")
+    y_data = np.random.rand(1, 10).astype("float32")
+    inputs = {"x": x_data, "y": y_data}
+    output_list = generate_ref_data(func, inputs)
+    input_list = [inputs["x"], inputs["y"]]
+    compile_and_run(func, input_list, output_list)
+
+
+def test_subtract():
+    i = relay.var("i", shape=[], dtype="int32")
+    sub = relay.subtract(i, relay.const(1, dtype="int32"))
+    func = relay.Function([i], sub, ret_type=relay.TensorType([], "int32"))
+    i_data = np.array(1, dtype="int32")
+    inputs = {"i": i_data}
+    output_list = generate_ref_data(func, inputs)
+    input_list = [inputs["i"]]
+    compile_and_run(func, input_list, output_list)
+
+
+def test_tuple_output():
+    x = relay.var("x", shape=(6, 9))
+    y = relay.split(x, 3).astuple()
+    a = relay.TupleGetItem(y, 0)
+    b = relay.TupleGetItem(y, 1)
+    c = relay.TupleGetItem(y, 2)
+    out = relay.Tuple([a, b])
+    func = relay.Function([x], out)
+    x_data = np.random.rand(6, 9).astype("float32")
+    inputs = {"x": x_data}
+    output_list = generate_ref_data(func, inputs)
+    input_list = [inputs["x"]]
+    compile_and_run(func, input_list, output_list)
+
+
+def test_mobilenet():
+    mod, params = testing.mobilenet.get_workload(batch_size=1)
+    data_shape = [int(x) for x in mod["main"].checked_type.arg_types[0].shape]
+    data = np.random.uniform(size=data_shape).astype("float32")
+    inputs = {"data": data}
+    output_list = generate_ref_data(mod, inputs, params)
+    input_list = [inputs["data"]]
+    compile_and_run(mod, input_list, output_list, params)
+
```

Comment:
Can we have the BYOC infra test : test_byoc_utvm that is in  tests/micro/zephyr/test_zephyr.py ?